main.js

```diff
@@ -0,0 +1,22 @@
+
+$('#submit').on('click', function() {
+  var text = $('#text').val();
+  var name = $('#name').val();
+
+  //comments appear above inputs
+
+$('.posts').append('<div class="post">' + text + " " + ' Posted By: ' + " " + name + '<button type="button" class="btn btn-link" id="option1" >Comment</button>' + '<button type="button" class="btn btn-link" id="option2" >Remove</button>');
+
+
+
+$('#option2').on('click', function () {
+  $('#here').remove();
```

Comment:
This ID removes the entire posts div, that's why it deletes everything and doesn't allow you to add posts after that, because the div no longer exists.